docs/detection/utils.md

```diff
@@ -18,16 +18,22 @@ status: new
 :::supervision.detection.utils.mask_iou_batch
```

Comment:
Let's move all docs related to non-max suppression and merge into separate docs file. Name the section `Double Detection Filter`.